cpu/lm4f120/periph/uart.c

```diff
@@ -28,6 +28,15 @@
 #include "periph_conf.h"
 
 /**
+ * @brief Unified interrupt handler for all UART devices
+ *
+ * @param uartnum       the number of the UART that triggered the ISR
+ * @param ulBase        the UART device that triggered the ISR
+ * @param ulRxInt       the RX Interrupt Mask of the UART Device that triggered the ISR
+ */
+static inline void irq_handler(uart_t uartnum, unsigned long ulBase, unsigned long ulRxInt);
```

Comment:
Here and all following, please don't use camel case. https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#naming